src/assets/wise5/common/save-indicator/save-indicator.component.html

```diff
@@ -0,0 +1,12 @@
+<div fxLayout="row wrap" fxLayoutAlign="start center">
+  <mat-spinner
+    *ngIf="globalMessage.isProgressIndicatorVisible"
+    class="spinner"
+    [mode]="'indeterminate'"
+    [diameter]="24"
```

Comment:
I don't think the 'spinner' class is ever used, is it? If not, can remove from here.